faculty/clients/experiment.py

```diff
@@ -55,6 +55,12 @@ class ExperimentRunStatus(Enum):
     ],
 )
 
+ExperimentRunMetric = namedtuple(
+    "ExperimentRunMetric", ["key", "value", "timestamp"]
+)
+ExperimentRunParam = namedtuple("ExperimentRunParam", ["key", "value"])
+ExperimentRunTag = namedtuple("ExperimentRunTag", ["key", "value"])
```

Comment:
Should these just be called `Metric`, `Param` and `Tag`?

Reply:
I think that's a good call